weaknesses/MASVS-NETWORK/MASWE-0052.md

```diff
@@ -40,7 +40,36 @@ draft:
   - MITM via an arbitrary certificate signed by a trusted CA works
   - WebView clients (e.g. WebViewClient.onReceivedSslError, not TLS errors ignored,
     mixed content, insecure handlers)
-status: draft
+status: new
 
 ---
 
+## Overview
+
+Applications that do not properly validate SSL/TLS certificates during secure communication are susceptible to man-in-the-middle attacks and other security breaches. This weakness occurs when an application accepts invalid, expired, self-signed, or untrusted certificates without appropriate verification, compromising the integrity and confidentiality of data in transit.
```

Comment:
We could change man-in-the-middle to machine-in-the-middle which is a common new phrase for it to be gender neutral.

Reply:
I was also thinking about that, I've seen also "Manipulator-in-the-middle" a few times. At the end of the day it's a threat actor and done manually in our scenarios, so maybe manipulator instead of machine. But I guess both would work and is better than "man".